packages/prelude/test/array-test.ts

```diff
@@ -43,6 +43,15 @@ describe('append', () => {
   })
 })
 
+describe('concat', () => {
+  it('should concat', () => {
+    assertSame(concat([1, 2], [3, 4]), [1, 2, 3, 4])
+    assertSame(concat([1, 2], []), [1, 2])
+    assertSame(concat([], [3, 4]), [3, 4])
+    assertSame(concat([], []), [])
```

Comment:
+1 for the left and right identity tests.

Since `concat` is the array semigroup, one minor suggestion is to add the usual associativity assertion, i.e. `(a + b) + c == a + (b + c)` (where `+` is concat).

Reply:
done